src/libs/ajax/Billing.js

```diff
@@ -78,11 +78,12 @@ export const Billing = signal => ({
    * @param billingProjectName
    * @param startDate, a string of the format YYYY-MM-DD, representing the start date of the report.
    * @param endDate a string of the format YYYY-MM-DD, representing the end date of the report.
+   * @param aggregationKeys a list of strings indicating how to aggregate spend data. subAggregation can be requested by separating keys with '~' e.g. 'Workspace~Category'
```

Comment:
Do we need an enum or some such thing to ensure the provided categories are valid?

Reply:
Rawls does validate the keys on its end. It could be more effort than it's worth to keep the validation in the UI up to date with what Rawls accepts, but it could also save us from making careless mistakes. I'm undecided, what do you think @aherbst-broad?